<?xml version="1.0"?>
<launch>

	<param name="detection_BoundingBoxes"  type="str"    value="/yolov5/detections" />
	<param name="depth_lidar_image"        type="str"    value="/depthImage" />
	<param name="depth_cam2_image"         type="str"    value="/camera2/aligned_depth_to_color/image_raw" />
	<param name="yoloImgTopic"        type="str"    value="/yoloImg_input" />
	<param name="odometryTopic"       type="str"    value="/odom" />

<!-- params -->
    <group ns="control_params">
        <rosparam file="$(find yolo_visual_servoing)/cfg/cfg_params.yaml" />
    </group>
    
   <group ns="matrix_file">
        <rosparam file="$(find lidar_camera_fusion)/cfg/cfg_params.yaml" />
    </group>

   <node pkg="yolo_visual_servoing" type="yoloVisualServo_node" name="yoloVisualServo_node" output="screen"/>

<include file="$(find yolo_visual_servoing)/launch/change_topic.launch" />
<include file="$(find pc2image)/launch/VLP16_image.launch" />
<include file="$(find odom_estimation_pc )/launch/odomEstimation_VLP16.launch" />
<include file="$(find lidar_camera_fusion)/launch/pcOnImage.launch" />

<!-- rviz- 
<node type="rviz" name="rviz" pkg="rviz" args="-d $(find lidar_camera_fusion)/rviz/lidar_camera.rviz" /> -->
  
</launch>

